kratos/includes/chunk.h

```diff
@@ -78,7 +78,7 @@ namespace Kratos
 	  }
 
       /// Destructor is not virtual. This class can not be drived.
-	  ~Chunk() noexcept {
+	  ~Chunk() noexcept override {
```

Comment:
This one is completely at odds with the comment just above. If the comment is right, this should not be an override (but may be a final). @pooyan-dadvand can you confirm?

Reply:
I agree with @jcotela and it should be a final!